contrib/examples/actions/mistral-handle-retry.yaml

```diff
@@ -0,0 +1,30 @@
+version: '2.0'
+name: "examples.mistral-handle-retry"
+
+workflows:
+
+    main:
+        type: direct
+        tasks:
+            task1:
```

Comment:
To make it fully automated, consider a parallel task with delay, which creates the file there.